content/ai/concepts/neural-networks/terms/convolutional-neural-networks/convolutional-neural-networks.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Convolutional Neural Networks'
+Description: 'Convolutional Neural Networks are a type of neural network that are primarily used for image classification.'
+Subjects:
+  - 'Computer Science'
+Subjects:
+  - 'Deep Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Neural Networks'
+  - 'CNN'
```

Comment:
```suggestion
```
CNN is not on the [tags list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).